### PR TITLE
latency, e2e: Build and use kiagnose and checkup images

### PR DESCRIPTION
**Pull request by orelmisan**
Build kiagnose and latency checkup's images from current source code, and use them in the e2e tests.

This PR is based on PR #75.

### CODE REVIEW

File: checkups/kubevirt-vm-latency/automation/e2e.sh [hunk 1]
```diff
@@ -50,6 +52,9 @@ while true; do
     --delete-cluster)
         OPT_DELETE_CLUSTER=1
         ;;
+    --build-kiagnose)
```
Comment (EdDev) [inline checkups/kubevirt-vm-latency/automation/e2e.sh:56]: Please do not introduce this target here. If it is needed, it should be done by the kiagnose script.

File: checkups/kubevirt-vm-latency/automation/e2e.sh [hunk 2]
```diff
@@ -126,8 +132,15 @@ if [ -n "${OPT_CREATE_CLUSTER}" ]; then
...
+if [ -n "${OPT_BUILD_KIAGNOSE}" ]; then
+  cd ${SCRIPT_PATH}/../../..
+  ./automation/make.sh --build-core --build-core-image
+  cd -
+fi
+
 if [ -n "${OPT_DEPLOY_KIAGNOSE}" ]; then
-    ${KUBECTL} apply -f manifests/kiagnose.yaml
+  ${KIND} load docker-image "${FRAMEWORK_IMAGE}" --name "${CLUSTER_NAME}"
+  ${KUBECTL} apply -f manifests/kiagnose.yaml
```
Comment (EdDev) [inline checkups/kubevirt-vm-latency/automation/e2e.sh:148]: nit: Can you use 4 spaced for indentation?

File: checkups/kubevirt-vm-latency/automation/e2e.sh [hunk 3]
```diff
@@ -97,10 +101,11 @@ if [ "${ARGCOUNT}" -eq "0" ] ; then
     OPT_BUILD_KIAGNOSE=1
     OPT_DEPLOY_KUBEVIRT=1
     OPT_DEPLOY_CNAO=1
+    OPT_BUILD_CHECKUP=1
     OPT_DEPLOY_CHECKUP=1
     OPT_DEFINE_NAD=1
     OPT_RUN_TEST=1
-    OPT_DELETE_CLUSTER=1
+#    OPT_DELETE_CLUSTER=1
```
Comment (EdDev) [inline checkups/kubevirt-vm-latency/automation/e2e.sh:108]: Leftover.

File: checkups/kubevirt-vm-latency/automation/e2e.sh [hunk 4]
```diff
@@ -211,11 +216,19 @@ EOF
 
 fi
 
+if [ -n "${OPT_BUILD_CHECKUP}" ]; then
+  cd ${SCRIPT_PATH}/..
+  ./automation/make.sh --build-checkup --build-checkup-image
+  cd -
```
Comment (EdDev) [inline checkups/kubevirt-vm-latency/automation/e2e.sh:222]: - Why can't you use `${PWD}`? It matters where it is ran from?
- Can you fix the indentation?

**Review by EdDev — CHANGES_REQUESTED** [review]
Thanks, see inline comments.

**Comment by orelmisan** [comment]
Work continues on PR #75.

**Pull request closed without merging**